VS/VehicleSignals.rdf

```diff
@@ -3185,6 +3189,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;LumbarPressureSensor">
 		<rdfs:subClassOf rdf:resource="&auto-vs;VehicleSensor"/>
+		<rdfs:label xml:lang="en">Lumbar pressure sensor</rdfs:label>
```

Comment:
Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.